webapp/components/comments/CommentList/index.vue

```diff
@@ -41,29 +40,5 @@ export default {
   props: {
     post: { type: Object, default: () => {} },
   },
-  data() {
-    return {
-      comments: [],
-    }
-  },
-  watch: {
-    Post(post) {
-      const [first] = post
-      this.comments = (first && first.comments) || []
-    },
-  },
-  apollo: {
-    Post: {
-      query() {
-        return PostCommentsQuery(this.$i18n)
-      },
-      variables() {
-        return {
-          slug: this.post.slug,
-        }
-      },
-      fetchPolicy: 'cache-and-network',
-    },
-  },
```

Comment:
Yesss, less code, less state, less bugs!

***State is the root of all evil***
![root of all evil](https://media.giphy.com/media/xl5QdxfNonh3q/giphy.gif)